Source/Core/DolphinQt/Config/GameConfigEdit.cpp

```diff
@@ -0,0 +1,170 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/Config/GameConfigEdit.h"
+
+#include <QCompleter>
+#include <QDesktopServices>
+#include <QFile>
+#include <QMenu>
+#include <QTextCursor>
+#include <QTextEdit>
+#include <QVBoxLayout>
+#include <QWhatsThis>
+
+#include "DolphinQt/Config/GameConfigHighlighter.h"
+
+GameConfigEdit::GameConfigEdit(QWidget* parent, const QString& path) : m_path(path)
+{
+  LoadFile();
+
+  setAcceptRichText(false);
+  setContextMenuPolicy(Qt::CustomContextMenu);
+
+  new GameConfigHighlighter(document());
+
+  m_keyword_map[QStringLiteral("Core")] =
+      tr("Section that contains most CPU- and Hardware related settings.");
+
+  m_keyword_map[QStringLiteral("CPUThread")] = tr(
+      "Controls whether or not Dual-Core should be enabled. Can improve performance but can also "
+      "cause issues. Defaults to <b>True</b>");
+  m_keyword_map[QStringLiteral("FastDiscspeed")] =
+      tr("Shortens loading times but may break some games. Defaults to <b>False</b>");
```

Comment:
This description should also mention that it decreases the performance, because that isn't obvious at all to the average user.